src/flows/pipes/RawFluxEditor/functions.tsx

```diff
@@ -48,57 +48,59 @@ const Functions: FC<Props> = ({onSelect}) => {
     [search]
   )
 
-  let fnComponent
+  return useMemo(() => {
```

Comment:
the diff looks crazy, but all I did was wrap a large section of this component in a `useMemo` which caused the indentation to change and then prettier adjusted some things too

Reply:
protip: add `?w=1` to the url to ignore whitespace

https://github.com/influxdata/ui/pull/1959/files?w=1

Reply:
DUDE

Reply:
![](https://media.giphy.com/media/1pA2TskF33668iVDaW/giphy.gif)